build.gradle.kts

```diff
@@ -42,13 +42,15 @@ detekt {
   baseline = file("$projectDir/detekt-baseline.xml") // a way of suppressing issues before introducing detekt
 }
 
+ext["hibernate.version"] = "6.5.3.Final"
```

Comment:
pinning hibernate as the latest version doesn't handle manually setting ids without use of a [custom sequence generator](https://hibernate.atlassian.net/browse/HHH-17472). Support for this will be included by default in hibernate 7